imports/ui/statistics/PlayerModal.js

```diff
@@ -0,0 +1,28 @@
+import Players from '../../api/Players';
+import Constants from '../../api/Constants';
+
+import './PlayerModal.html';
+
+Template.PlayerModal.helpers({
+	getPlayerInfo() {
+		let player = Session.get("selectedStatistics");
+		if (player) {
+			let info = {};
+			info.name = player.leagueName;
+			info.elo = player.elo.toFixed(3);
+			info.winRate = (player.handsWin / player.handsTotal * 100).toFixed(1);
```

Comment:
It takes up a bit of extra room unfortunately... which makes it look a tad worse on mobile. Maybe if Casper decides that he wants it later we can, but I'd like to leave it at one decimal so it's a bit more comfortable.